packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/list/template.html

```diff
@@ -0,0 +1,112 @@
+<oui-datagrid class="table-striped" data-rows="$ctrl.listeners">
+    <oui-datagrid-topbar>
+        <oui-button
+            data-ng-click="$ctrl.goToListenerCreation()"
+            variant="secondary"
+            icon-left="oui-icon-plus"
+        >
+            <span
+                data-translate="octavia_load_balancer_listeners_add_button"
+            ></span>
+        </oui-button>
+    </oui-datagrid-topbar>
+
+    <oui-datagrid-column
+        data-title=":: 'octavia_load_balancer_listeners_name' | translate"
+        data-property="name"
+        data-sortable
+        data-searchable
+        data-filterable
+        data-type="string"
+        data-prevent-customization
+    >
+        <a data-ng-click="$ctrl.goToListener($row)">
+            {{ ::$value }}
+        </a>
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title=":: 'octavia_load_balancer_listeners_default_pool' | translate"
+        data-property="defaultPoolId"
+        data-sortable
+        data-searchable
+        data-filterable
+        data-type="string"
+        data-helper=":: 'octavia_load_balancer_listeners_default_pool_helper' | translate"
+    >
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title=":: 'octavia_load_balancer_listeners_protocol' | translate"
+        data-property="protocol"
+        data-sortable
+        data-searchable
+        data-filterable
+        data-type="string"
+        data-helper=":: 'octavia_load_balancer_listeners_protocol_helper' | translate"
+    >
+    </oui-datagrid-column>
+    <!--
+    <oui-datagrid-column
+        data-title=":: 'octavia_load_balancer_listeners_certificate' | translate"
+        data-property="createdAt"
+        data-sortable
+        data-searchable
+        data-filterable
+        data-type="date"
+    >
+    </oui-datagrid-column>
+    -->
```

Comment:
```suggestion
```

Reply:
It will be part of batch 2 and I was thinking that it would be acceptable to do it now but I can clean indeed